ecl/hql/hqlexpr.cpp

```diff
@@ -6349,6 +6346,8 @@ void CHqlDataset::cacheParent()
     case no_mergejoin:
     case no_nwayjoin:
     case no_nwaymerge:
+    case no_serialize:
+    case no_deserialize:
         rootTable = this;       //?
```

Comment:
@ghalliday whilst here might it be worth addressing this comment "//?" or at least expanding on it - or perhaps open another jira?